identity-apps-core/apps/authentication-portal/src/main/webapp/includes/analytics.jsp

```diff
@@ -1,10 +1,9 @@
 <script>
 /*
- * @param eventName - name of the event fired
- * @param metadataObj - associated metadata in form of JS object
+ * @param {string} eventName - name of the event fired
+ * @param {object} metadataObj - associated metadata in form of JS object
  */
 function trackEvent (eventName, metadataObj) {
```

Comment:
Since this function is now an empty function, we can remove it, right?

Reply:
This function is a placeholder function for the analytics extension point. It is there for anyone to add their own event publishing logic. Removing this can cause compilation errors, therefore it's possible to remove this.